ui/raidboss/data/04-sb/raid/o12n.js

```diff
@@ -19,10 +19,10 @@
   triggers: [
     {
       id: 'O12N Solar Ray',
-      regex: / 14:(?:330F|3310):(?:Omega|Omega-M) starts using (?:Unknown_330F|Unknown_3310|Solar Ray) on (\y{Name})/,
-      regexDe: / 14:(?:330F|3310):(?:Omega|Omega-M) starts using (?:Unknown_330F|Unknown_3310|Sonnenstrahl) on (\y{Name})/,
-      regexFr: / 14:(?:330F|3310):(?:Oméga|Oméga-M) starts using (?:Unknown_330F|Unknown_3310|Rayon Solaire) on (\y{Name})/,
-      regexJa: / 14:(?:330F|3310):(?:オメガ|オメガM) starts using (?:Unknown_330F|Unknown_3310|ソーラレイ) on (\y{Name})/,
+      regex: / 14:(?:330F|3310):(?:Omega|Omega-M) starts using Solar Ray/,
+      regexDe: / 14:(?:330F|3310):(?:Omega|Omega-M) starts using Sonnenstrahl/,
+      regexFr: / 14:(?:330F|3310):(?:Oméga|Oméga-M) starts using Rayon Solaire/,
+      regexJa: / 14:(?:330F|3310):(?:オメガ|オメガM) starts using ソーラレイ/,
```

Comment:
These lines lost the `on (\y{Name})` capturing group.

Reply:
done :)